src/Symfony/Component/Security/Http/Authenticator/AccessTokenAuthenticator.php

```diff
@@ -97,7 +91,7 @@ public function onAuthenticationFailure(Request $request, AuthenticationExceptio
 
     public function setTranslator(?TranslatorInterface $translator)
     {
-        $this->translator = $translator;
+        trigger_deprecation('symfony/security-http', '6.2', '"%s()" has no effect and will be removed in 7.0.', __METHOD__);
```

Comment:
I would postpone the runtime deprecation to 6.4. We usually don't deprecate stuff in bugfix releases.

Reply:
So, no-op for 6.2 and adding `trigger_deprecation` in 6.4?

Reply:
WDYT of just not wiring this on 6.2 (aka only patching the bundle) and no-op + deprec go on 6.4?

Reply:
or better: we keep the translator in place, no need to change the behavior in 6.*, unwiring is enough to fix the issue

Reply:
I still suggest to deprecate the method in 6.4 to be able to remove it in 7.0